python_scripts/02_numerical_pipeline_hands_on.py

```diff
@@ -218,8 +218,11 @@
 print(f"Accuracy of logistic regression: {accuracy:.3f}")
 
 # %% [markdown]
-# Now the real question is: is this generalization performance relevant of a good
-# predictive model? Find out by solving the next exercise!
+# When calling the `score` method on a classification problem, the output by
+# default is the fraction of correctly classified samples. In this case, around
```

Comment:
I think it is weird to say "when calling the `score` method ... the output by default", there is no way to change what `score` returns:

Something like this maybe (the wording can probably be improved):
```suggestion
# In scikit-learn, the `score` method of a classification model returns the accuracy,
# i.e. the fraction of correctly classified samples. In this case, around
```